File: Remove Duplicate from Sorted Array/Remove_Duplicate_from_Sorted_Array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int i =0,j = 1;
        int n = nums.size();
        while(j<n) {
            if(nums[i] == nums[j]) j++;
            else {
                nums[i+1] = nums[j];
                j++;
                i++;
            }
        }
        return i+1;
    }
};

File: Remove Duplicate from Sorted Array/Remove_Duplicate_from_Sorted_Array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i, j = 0, 1
        n = len(nums)
        while j < n:
            if nums[i] == nums[j]:
                j += 1
            else:
                nums[i + 1] = nums[j]
                i += 1
                j += 1
        return i + 1
        